couchdb/tests/loader.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Daniel Holth
+# All rights reserved.
+#
+# This software is licensed as described in the file COPYING, which
+# you should have received as part of this distribution.
+
+import unittest
+import os.path
+
+from couchdb import loader
+from couchdb.tests import testutil
+
+# loaded by couchdbkit with _attachments and couchapp keys removed:
```

Comment:
I reference couchdbkit because I want to prove to myself that this code can load exactly the same data as couchdbkit. According to the test, it does!

I think I forgot to add this test to the list of all couchdb-python tests.

Sorry @iblis17 couchapp is not Python 3 compatible is it? Also my environment is entirely scripted and I need the Python API, and I use couchdb-python extensively.